src/Command/Generate/PluginConditionCommand.php

```diff
@@ -97,9 +97,9 @@ protected function interact(InputInterface $input, OutputInterface $output)
     {
         $io = new DrupalStyle($input, $output);
 
-        $entity_manager = $this->getEntityManager();
+        $entityTypeRepository = $this->getService('entity_type.repository');
```

Comment:
fine we can rely on the drupal core services name b/c they are not going to remove before 9.x. let's see if @jmolivas would be happy here.